# GOAL
string locationName
---
# RESULT
bool success
---
# FEEDBACK
Position currentPosition
